docs/reference/commandline/inspect.md

```diff
@@ -63,7 +63,7 @@ $ docker inspect --format='{{.LogPath}}' $INSTANCE_ID
 ### Get an instance's image name
```

Comment:
Let me know what changes you'd like me to make. Also feel free to amend this commit if that's easier.